apps/src/openni_mls_smoothing.cpp

```diff
@@ -37,12 +37,12 @@
 #include <pcl/console/parse.h>
 #include <pcl/io/openni_camera/openni_driver.h>
 #include <pcl/io/openni_grabber.h>
+#include <pcl/search/kdtree.h> // for KdTree
 #include <pcl/surface/mls.h>
```

Comment:
Any reason comments aren't aligned here, when they are in apps/include/pcl/apps/impl/dominant_plane_segmentation.hpp?

Reply:
I believe Clang-Format does this only, when there are comments in successive lines